### PR TITLE
DT-1949: ⬆️ Update to latest localstack

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: docker-compose-all.yml [hunk 1]
```diff
@@ -96,13 +96,12 @@ services:
...
   localstack:
-    # pinned to 10.4 due to https://github.com/localstack/localstack/issues/1992
-    image: localstack/localstack:0.10.4
+    image: localstack/localstack:0.12.9.1
     networks:
       - hmpps
     container_name: localstack
     ports:
-      - "4567-4597:4567-4597"
+      - "4566-4597:4566-4597"
```
Comment (sp-watson) [inline docker-compose-all.yml:104]: Why the change of ports?

Comment (mikehalmamoj) [reply docker-compose-all.yml:104]: Rather than a port for each service they have consolidated all services on the same port.

**Review by mikehalmamoj — APPROVED** [review]
